packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -6,18 +6,50 @@ import { axiosBaseQuery } from './axios';
 
 import type { CreateRelease, GetReleases } from '../../../shared/contracts/releases';
 
+export interface GetAllReleasesQueryParams {
+  page?: number;
+  pageSize?: number;
+  filters?: {
+    $and?: Array<{
+      releasedAt?: {
+        $notNull?: boolean | 'true' | 'false';
+      };
+    }>;
+  };
+}
+
 const releaseApi = createApi({
   reducerPath: pluginId,
   baseQuery: axiosBaseQuery,
   tagTypes: ['Releases'],
   endpoints: (build) => {
     return {
-      getRelease: build.query<GetReleases.Response, undefined>({
-        query() {
+      getReleases: build.query<GetReleases.Response, GetAllReleasesQueryParams | void>({
+        query({ page, pageSize, filters } = { page: 1, pageSize: 16, filters: undefined }) {
           return {
             url: '/content-releases',
             method: 'GET',
+            config: {
+              params: {
+                page,
+                pageSize,
+                filters,
+              },
+            },
+          };
+        },
+        transformResponse(response: GetReleases.Response, meta, arg) {
+          const releasedAtValue = arg?.filters?.$and?.[0]?.releasedAt?.$notNull;
+          const isActiveDoneTab = releasedAtValue === 'true';
+          const newResponse = {
+            ...response,
+            meta: {
+              ...response.meta,
+              activeTab: isActiveDoneTab ? 'done' : 'pending',
+            },
           };
+
+          return newResponse;
         },
```

Comment:
This is really necessary? You can get the selected tab by the query param in the ReleasesPage field, I will let a comment with what I mean